### PR TITLE
Added test common methods to the super class and Fix the issue with sample

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import javax.xml.xpath.XPathExpressionException;
 import java.net.MalformedURLException;
 import java.util.Stack;
+import java.util.concurrent.TimeUnit;
 
 public abstract class DSUIIntegrationTest extends DSIntegrationTest {
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Add class level comments.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 2]
```diff
@@ -134,11 +138,11 @@ public void popWindow() throws MalformedURLException, XPathExpressionException {
      * @param pwd      password
      * @throws javax.xml.xpath.XPathExpressionException,InterruptedException
      */
-    public void login(String userName, String pwd)
-            throws Exception {
+    public void login(String userName, String pwd) throws Exception {
         String fullUrl = "";
         fullUrl = getBaseUrl() + DS_SUFFIX;
         driver = getDriver();
+
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Remove all unnecessary blank lines throughout the code.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 3]
```diff
@@ -134,11 +138,11 @@ public void popWindow() throws MalformedURLException, XPathExpressionException {
      * @param pwd      password
      * @throws javax.xml.xpath.XPathExpressionException,InterruptedException
      */
-    public void login(String userName, String pwd)
-            throws Exception {
+    public void login(String userName, String pwd) throws Exception {
         String fullUrl = "";
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Unnecessary assignment, could have merged this line with next line, and directly initialize with getBaseUrl() + DS_SUFFIX.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 4]
```diff
@@ -152,11 +156,11 @@ public void login(String userName, String pwd)
      *
      * @throws javax.xml.xpath.XPathExpressionException
      */
-    public void logout() throws
-            Exception {
+    public void logout() throws Exception {
         String fullUrl = "";
         fullUrl = getBaseUrl() + DS_HOME_SUFFIX;
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Same issue as explained above.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java [hunk 5]
```diff
@@ -197,17 +202,80 @@ public void logoutFromAdminConsole() throws Exception {
      */
     public void addDashBoard(String dashBoardTitle, String description) throws Exception {
         driver = getDriver();
-        driver.get(getBaseUrl()+"/portal/dashboards");
+
+        redirectToLocation("portal", "dashboards");
         driver.findElement(By.cssSelector("[href='create-dashboard']")).click();
         driver.findElement(By.id("ues-dashboard-title")).clear();
```
Comment (sinthuja) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: If these Ids are being used in multiple places and especially multiple classes, add them as constants and reuse them without declaring everywhere.

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 1 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 2 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 3 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 4 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java (hunk 5 above)
Comment (NipunaMarcus) [inline modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

**Pull request merged**